CONTRIBUTING.md

```diff
@@ -13,53 +13,114 @@ At the same time, I had a day job where I was a junior software developer at a r
-- **Instant dev cred:** I was accepted to give my first public talk at a conference based solely on my open source work. It was a terrible talk, but who's first talk is good!? :D
+- **Instant dev cred:** I was accepted to give my first public talk at a conference based solely on my open source work. It was a terrible talk, but whose first talk is good!? :D
 - **Leadership/mentorship opportunities:** I went from having zero responsibility at work to being a respected voice/opinion in the `request` GitHub issues and PRs.
 - **Learning from smart people:** I got to meet and learn from so many smart people across the open source ecosystem.
-- **preventing imposter syndrome:** Sure, I was still just a kid, but having an actual human connection to developers who I looked up to at the time helped dispell the idea that "oh, **I** could never be like that."
+- **preventing imposter syndrome:** Sure, I was still just a kid, but having an actual human connection to developers who I looked up to at the time helped dispell the idea that "oh, _I_ could never be like that."
 - **Making friends in the larger community:** The creator of request, [@mikeal](https://twitter.com/mikeal), is still a friend to this day.
 
-If any of this sounds interesting, I hope you consider getting involved with Astro. Come say hi in the [**#new-contributors**](https://astro.build/chat) channel on Discord, anytime. We're always around and value contributions of any shape/size.
+If any of this sounds interesting, I hope you consider getting involved with Astro. Come say hi in the [**#new-contributors**][discord] channel on Discord anytime. We're always around and value contributions of any shape/size.
 
 # Contributor Manual
 
 ## Prerequisite
 
-```shell
-node: "^12.20.0 || ^14.13.1 || >=16.0.0"
-yarn: "^1.22.10"
-# otherwise, your build will fail
-```
+You’ll need the following tools set up on your system to contribute to this codebase:
 
-## Setting up your local repo
+- [Node 14.x or 16.x][install-node]
+- [Yarn 1.x][install-yarn] (_Note: we do NOT use Yarn 2!_)
+- [Git 2.x][install-git]
 
-Astro uses yarn workspaces, so you should **always run `yarn install` from the top-level project directory.** running `yarn install` in the top-level project root will install dependencies for `astro`, `www`, `docs`, and every package in the repo.
+## Setting up Astro locally
 
-```shell
-git clone && cd ...
-yarn install
-yarn build:all
+All work on Astro begins by downloading a copy locally. Start by [**forking**][how-to-fork] this repository (if you’re new to this, [GitHub has a great detailed guide][how-to-fork]). You should end up with an `astro/` code folder on your local machine.
+
+Switch to this folder by opening your **Terminal application** of choice and running:
+
+```
+cd /[path to my code folder]/astro
 ```
 
+Next, you’ll need to install dependencies and set up the project. You can do that by running the following commands from within the `astro/` folder:
```

Comment:
Looks like all references have been removed so many a non-issue, but I liked the idea of defining "top-level project root" as a concept / term here so that we could use it later in the document.

"`astro/` folder" is confusing since it could also refer to the `astro/` folder inside of `packages/`.